src/main/resources/application-dev.properties

```diff
@@ -10,3 +10,5 @@ spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.open-in-view=false
 spring.graphql.graphiql.enabled=true
+
```

Comment:
Remove these lines and the file shouldn't show up as changed. Less code to review.